packages/solo/src/main.js

```diff
@@ -1,3 +1,4 @@
+/* global solo */
```

Comment:
`solo` is not a global. It's defined in this file.

Somehow the conversion to arrow functions changed `export default async function solo() { ... }` to an anonymous `export default const async () => { ... }`

Reply:
We don't want that! Any alternative solutions you would recommend?